manpages/lumina-open.8

```diff
@@ -0,0 +1,106 @@
+.Dd November 6, 2017
+.Dt LUMINA-OPEN 1
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-open
+.Nd a graphical front-end to xdg-open.
+It is used to open files or links with the proper application.
+
+.Sh SYNOPSIS
+.Nm
+.Pp
+.Nm
+.Op Fl select | action Ar ActionID
+.Op Ar FILE_PATH | URL
+.Pp
+.Nm
+.Cm [-volumeup | -volumedown | -brightnessup | -brightnessdown |
+.Cm -autostart-apps | -terminal]
+
+.Sh DESCRIPTION
+Given a file with an absolute pathway or URL, the
+.Nm
+utility finds the appropriate application with which to open the
+file. If the file is a *.desktop application shortcut,
+.Nm
+starts the application automatically. Using the appropriate flags
+allows
+.Nm
+to perform a few specific system operations. Here are the commands,
+flags, and descriptions of their functionality:
```

Comment:
Suggested rewrite (2 sentences starting with "Using" and ending with "functionality"):
.Nm
can also perform specific system operations.
Here are all flags and their descriptions: